### ISSUE TITLE
Notification bar elements are missing after upgrading to RC1.

### ISSUE THREAD

**Issue by Mishasama** [guardianproject/orbot#940]
Why remove the display IP information and Snowflake count from the notification bar?
It is very helpful for understanding the country you are currently in. And it is also convenient to see whether the number of people helped has increased.

**Comment by tkbstudios** [comment]
Yeah I think the devs should add this back at the bottom of the home screen after Orbot is running to choose exit node location and see in which country we are like in the old UI

**Comment by bitmold** [comment]
Snowflake proxy with a persistent notification in newer releases. Closing this old ticket
